proposals/extensions.md

```diff
@@ -1125,36 +1142,40 @@ of a class, struct, interface, /***or extension** type `T`, and `A` is an *argum
 
 ### Operators
 
-TODO
+TODO(static)
 User-defined conversion should be allowed, except where it conflicts with a built-in
 conversion (such as with an underlying type).  
 
 ### Collection initializers
 
-TODO
+TODO(instance)
 https://github.com/dotnet/csharpstandard/blob/draft-v7/standard/expressions.md#128164-collection-initializers
 Explain how extension types factor in when resolving `Add` calls.
 
 ### Method group conversions
 
+TODO4 spec this section
+
 https://github.com/dotnet/csharpstandard/blob/draft-v7/standard/conversions.md#108-method-group-conversions
 TODO A single method is selected corresponding to a method invocation, 
 but with some tweaks related to normal form and optional parameters.  
-TODO There's also the scenario where a method group contains a single method (lambda improvements).  
-TODO There's also the scenario where method groups have a natural type even in the presence
-  of multiple extension methods, and even though the scenario remains an error for other reasons.
-  > A method group has a natural type if all candidate methods in the method group have a common signature.
-    (If the method group may include extension methods, the candidates include the containing type and all extension method scopes.)
 
 ### Pattern-based invocations and member access
 
-TODO Need to scan through all pattern-based rules for known members to consider whether to include extension type members.
+TODO(static) Need to scan through all pattern-based rules for known members to consider whether to include extension type members.
   If extension methods were already included, then we should certainly include extension type methods.
   Otherwise, we should consider it (for example `Current` property in `foreach`).
```

Comment:
Looks like this example is unnecessary since there's now a whole list of examples.

```suggestion
  Otherwise, we should consider it:
```